caddyconfig/caddyfile/dispenser.go

```diff
@@ -201,6 +202,43 @@ func (d *Dispenser) Val() string {
 	return d.tokens[d.cursor].Text
 }
 
+// RawVal gets the raw text of the current token (including quotes).
+// If there is no token loaded, it returns empty string.
+func (d *Dispenser) RawVal() string {
```

Comment:
Should we name this ValRaw for symmetry? 🤷‍♂️

Reply:
Hmm, yes I think that makes sense!